integration/kube_integration_test.go

```
@@ -1233,7 +1233,7 @@ func runKubeDisconnectTest(t *testing.T, suite *KubeSuite, tc disconnectTestCase
 	sessionCtx, sessionCancel := context.WithCancel(ctx)
 	go func() {
 		defer sessionCancel()
-		err := kubeExec(proxyClientConfig, execInContainer, kubeExecArgs{
+		kubeExec(proxyClientConfig, execInContainer, kubeExecArgs{
```

Why swallow the error?